lib/opensusebasetest.pm

```diff
@@ -1394,6 +1394,19 @@ sub post_fail_hook {
     send_key 'spc';
 
     $self->export_logs;
+
+    if (is_public_cloud() || is_openstack) {
```

Comment:
For consistency:
`if (is_public_cloud() || is_openstack()) {`
or
`if (is_public_cloud || is_openstack) {`